src/rust/engine/process_execution/src/lib.rs

```diff
@@ -546,9 +546,7 @@ pub struct Process {
 
   pub execution_strategy: ProcessExecutionStrategy,
 
-  /// Properties used for remote execution configuration. Regardless of remote execution, these
-  /// should impact the cache key.
-  pub platform_properties: Vec<(String, String)>,
+  pub remote_execution_platform_properties: Vec<(String, String)>,
```

Comment:
@tdyas @stuhood Rust question: I could move this into `ProcessExecutionStrategy::RemoteExecution` for clearer modeling. But then we don't have a meaningful name, just `RemoteExecution(Vec<(String, String)>)`. So I could use a type alias and/or newtype struct.

Thoughts on what's better?

Reply:
Yea, I do tend to stick to "tuple structs" or "tuple variants" in `enums` longer than I used to. I wouldn't worry about converting it from a tuple to a full struct until/unless you get two similarly-typed fields which might be confused for one another. A comment above the variant is fine.

Note also though that you can do:
```rust
enum SomeEnum {
    NoData,
    SomeData {
      data: Vec<(String,String)>,
    },
}
```
...which can be a bit easier than defining a newtype iff you don't actually ever need to use the newtype outside the context of the enum.